lib/dr_rockter/md.rb

```diff
@@ -3,15 +3,15 @@
 
 module DrRockter
   DeserializerError = Class.new(StandardError)
-  
+
```

Comment:
I'm not against stripping extra whitespaces (though I **do** like them), but could you do it in a separate PR so as not to pollute this one?

Reply:
👍